webapp/src/main/webapp/vue-apps/matrix/main.js

```diff
@@ -61,7 +62,8 @@ const components = {
   'message-receipt-list': MessageReceiptList,
   'message-receipt': MessageReceipt,
   'message-read-receipt-list-drawer': MessageReadReceiptListDrawer,
-  'message-typing-indicator': MessageTypingIndicator
+  'message-typing-indicator': MessageTypingIndicator,
+  'voice-message-recorder': VoiceMessageRecorder
```

Comment:
We should rename components to avoid collision by adding a prefix `matrix-` (not `meeds-` neither)